v1/backend/JAVA_LV_0_잼민_펀치/src/main/java/com/example/maddaewoole/Main.java

```diff
@@ -3,6 +3,22 @@
 public class Main {
 
 	public static void main(String[] args) {
+		Jpark2 jpark2 = new Jpark2();
+		Daewoole daewoole = new Daewoole();
+		final String PRINT_MENT = "지원은 '%s'를 시전하여 대욱의 분노를 %d 중가시켰다.\n";
 
+		while (daewoole.getPunchFlag()) {
+			String ment;
+			int prevokePoint;
+
+			ment = jpark2.prevoke();
+			prevokePoint = daewoole.bePrevoked(ment);
```

Comment:
```suggestion
			String ment = jpark2.prevoke();
			int prevokePoint = daewoole.bePrevoked(ment);
```

선언과 동시에 할당을 하는 것이 권장돼요